test/infrastructure/docker/e2e/docker_suite_test.go

```diff
@@ -104,11 +124,14 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	Expect(writeLogs(mgmt, "capi-system", "capi-controller-manager", logPath)).To(Succeed())
+	Expect(writeLogs(mgmt, "capd-system", "capd-controller-manager", logPath)).To(Succeed())
```

Comment:
I did something similar to this with the aws provider at first (just dumping the logs in AfterSuite), but you can actually lose the logs this way if the controller crashes for any reason, which is what led me down trying to stream the logs.

Reply:
ah ok, thanks for the pointer